src/diagnostics/MOM_sum_output.F90

```diff
@@ -1046,8 +1046,8 @@ subroutine accumulate_net_input(fluxes, sfc_state, tv, dt, G, US, CS)
     if (associated(fluxes%lprec) .and. associated(fluxes%fprec)) then
       do j=js,je ; do i=is,ie
         FW_in(i,j) = RZL2_to_kg * dt*G%areaT(i,j)*(fluxes%evap(i,j) + &
-            (((fluxes%lprec(i,j) + fluxes%vprec(i,j)) + fluxes%lrunoff(i,j)) + &
-              (fluxes%fprec(i,j) + fluxes%frunoff(i,j))))
+            (((fluxes%lprec(i,j) + fluxes%vprec(i,j)) + fluxes%lrunoff(i,j) + fluxes%lrunoff_glc(i,j)) + &
```

Comment:
Please add parentheses to specify the order of the sums with these new terms.

Reply:
Addressed in 69220e9cebf8bfdade695c9e617fd6002d99c84a.